test/lib/cfn-example.json

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
Is this just magically picked up by the test suite or is this a test file?

Reply:
No, commited by accident. I deleted it, thank you!